src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const longDays = 7;
+  const shortDays = 3;
+  const longDiscount = 50;
+  const shortDiscount = 20;
+  const price = cost * days;
+
+  if (days >= longDays) {
+    return price - longDiscount;
```

Comment:
Use 'return price - LONG_TERM_DISCOUNT;' directly without the 'else' block, as the 'return' statement will exit the function immediately if the condition is true.